app/gui2/src/components/GraphEditor/GraphVisualization.vue

```diff
@@ -51,41 +51,29 @@ const expressionInfo = computed(
 )
 const typeName = computed(() => expressionInfo.value?.typename ?? 'Any')
 
-/** The Visualization ID for the method to get the default visualization. */
-const defaultVisualizationVisualizationId = random.uuidv4() as Uuid
-const defaultVisualization = computedAsync(async () => {
-  if (props.currentType) return
-  if (!props.expressionId) return
-  const ident = props.pattern?.repr()
-  if (!ident) return
-  const ls = await projectStore.lsRpcConnection
-  const dataServer = await projectStore.dataConnection
-  ls.executeExpression(
-    projectStore.executionContext.id,
-    defaultVisualizationVisualizationId,
-    props.nodeId as ExprId,
-    `${ident}.default_visualization.to_js_object.to_json`,
-  )
-  return new Promise<VisualizationIdentifier | undefined>((resolve) => {
-    const onVisualizationUpdate = (payload: VisualizationUpdate, uuid: Uuid | null) => {
-      if (uuid !== defaultVisualizationVisualizationId) return
-      dataServer.off(`${OutboundPayload.VISUALIZATION_UPDATE}`, onVisualizationUpdate)
-      const data = payload.dataString()
-      const parsed: { library: { name: string } | null; name: string } | undefined = data
-        ? JSON.parse(data)
-        : undefined
-      resolve(
-        parsed && {
-          name: parsed.name,
-          module:
-            parsed.library == null
-              ? { kind: 'Builtin' }
-              : { kind: 'Library', name: parsed.library.name },
-        },
-      )
+const configForGettingDefaultVisualization = computed<NodeVisualizationConfiguration | undefined>(
+  () => {
+    if (props.currentType) return
+    if (!props.expressionId) return
+    return {
+      visualizationModule: 'Standard.Visualization.Helpers',
+      expression: 'a -> a.default_visualization.to_js_object.to_json',
```

Comment:
This should also work, but I leave it up to you.
```suggestion
      expression: '_.default_visualization.to_js_object.to_json',
```

Reply:
i do prefer that, but i tested just now and... unfortunately that seems to not work - with a `visualizationEvaluationFailed` error saying "Message not supported."